docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -145,35 +149,34 @@ Both offer monitoring of a single process instance, its variables and path, inci
 
 ## Limitations and availability
 
-:::note
-Play is being rebuilt and progressively rolled out to more users. This section explains why you might not see the **Play** tab.
-:::
+Play is being rebuilt and progressively rolled out to more users. This section explains why you might not see the **Play** tab, and any additional limitations.
+
+For more information about terms, refer to our [licensing and terms page](https://legal.camunda.com/licensing-and-other-legal-terms#c8-saas-trial-edition-and-free-tier-edition-terms).
+
+### Camunda 8 SaaS
 
-For Camunda 8 SaaS, Play is available to all Web Modeler users with editor or admin permissions within a project.
+In Camunda 8 SaaS, Play is available to all Web Modeler users with editor or admin permissions within a project.
 Enterprise users need an admin to enable Play by opting in to [alpha features](/components/console/manage-organization/enable-alpha-features.md).
 
-For Self-Managed, Play is controlled by the **PLAY_ENABLED** configuration property in Web Modeler. It is `true` by default for the Docker and Kubernetes distributions.
+### Camunda 8 Self-Managed
 
 :::note
-[Decision table rule](/components/modeler/dmn/decision-table-rule.md) evaluations are not viewable from Play. However, they can be inferred from the output variable, or can be viewed from Operate.
+To use Play with Docker, ensure OAuth is enabled for your configured components. The `docker-compose-core.yaml` file in the Camunda [platform repository](https://github.com/camunda/camunda-platform) does not provide authentication, and cannot be used with Play.
 :::
 
-For more information about terms, refer to our [licensing and terms page](https://legal.camunda.com/licensing-and-other-legal-terms#c8-saas-trial-edition-and-free-tier-edition-terms).
+In Self-Managed, Play is controlled by the `PLAY_ENABLED` [configuration property](/self-managed/modeler/web-modeler/configuration/configuration.md#feature-flags) in Web Modeler. This is `true` by default for the 8.6 Docker and Kubernetes distributions, and can be accessed as an alpha feature in 8.6.0-alpha5.
 
-## Configuration for Self-Managed
+Alpha features can be accessed by installing the alpha [Helm charts](https://github.com/camunda/camunda-platform-helm/tree/main/charts/camunda-platform-alpha), or running the alpha [Docker Compose](https://github.com/camunda/camunda-platform/tree/main/docker-compose/camunda-8.6) configuration.
```

Comment:
I restructured this content a bit. Unlike enabling alpha features for SaaS, we don't (yet?) have a guide on enabling/accessing this content yet for SM, or I think it would be appropriate to link here. 

I don't think detailed configuration instructions are necessary if this feature is enabled by default in the versions it's available in. There should be no need to change the `PLAY_ENABLED` flag unless you want to turn it off, in which case you can read about SM configuration on the linked page. 

I also don't think we should/need to provide specific alpha download instructions in their own section. Upgrading to the latest release, or choosing to specifically access the alpha feature early using the linked distributions, seems like it would be enough (for this page) to let SM users know how to opt in early.

Reply:
@conceptualshark here are the different potential setups
- self managed modeler, sass cluster
- docker self managed, full platform installation
-docker self managed web modeler with C8 run 
- alpha helm charts

The idea was to lean on docs for the 8.6 release to make the setup journey less painful. Can you help ensure the experience is good?

Reply:
I am happy to provide support on this experience for 8.6, but I think that's separate to what is required as an update for alpha5, which is this PR's current target. What is the expectation around docs delivery and experience for the upcoming alpha? The SM experience now seems limited to installing the alpha Helm chart, or accessing the alpha docker compose file, so I'm not sure what could be provided in this PR to improve that experience.

My expectation was that this feature would be out of alpha and added to the standard Helm and Docker configurations in 8.6, and so anyone using the latest docker, Helm, or upgrading existing Helm charts would have access to Play automatically (eliminating the need for any additional setup). For anyone needing to update existing Docker configurations, we already reference platform repo for examples in the [SM docs](https://docs.camunda.io/docs/next/self-managed/setup/deploy/local/docker-compose/#web-modeler).

Reply:
> My expectation was that this feature would be out of alpha and added to the standard Helm and Docker configurations in 8.6, 
Yes, this should be the case. Should we leave the reference to alpha5 in the docs until then and create a separate PR to bump this to 8.6 when the time comes?

Reply:
@yT0n1 👍 This is my opinion as well - I updated the PR to make the alpha SM requirements a little more clear. As we get closer to 8.6, we can revisit this page for any updates/deletions/additional options required. 

@HanselIdes That said, happy to update this further to support product. I assumed it would be convenient to have the SM alpha instructions referenced in this page for any testing, and can follow-up to remove alpha references before 8.6. If we'd prefer not to have them at all here, just let me know.

Reply:
> What is the expectation around docs delivery and experience for the upcoming alpha?

@conceptualshark I care that I can reference eager customers to working documentation starting with 8.6-alpha5 and that the installation journey is streamlined for the 8.6.0 release. The plan of merging this now and then following up with improved installation instructions for 8.6.0 makes sense to me

It seems like you and @yT0n1 have a good plan, so I'll add my approval and we can wrap this up